src/main/java/firebase/sdk/dashboard/dao/UserDao.java

```diff
@@ -15,8 +15,7 @@
    * @param id The unique id that identifies the user.
    * @return A User object containg data that represents the user and their favorite sdks.
    */
-  public User getUser(String id);
-
+  
```

Comment:
Why do we remove getUser?

Reply:
I did not mean to include this file in the pr sorry